dockers/docker-pde/stress-sfp-i2c.py

```diff
@@ -151,4 +155,17 @@ def stress_sfp_i2c(sec=180):
     sec = 180
     if len(sys.argv) >= 2:
         sec = int(sys.argv[1])
+
+
+
+
+
+
+
```

Comment:
Why add all these newlines?

Reply:
Looks like caused by removing conflicts, I can remove this